### PR TITLE
Magnetorquer factory

### PR DESCRIPTION
**Pull request by shaiTheKimhi**

### CODE REVIEW

File: Modules/Magnetorquer/fake_magnetorquer.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class fake_magnetorquer:
+    """An empty object representing magnetorquer""""
+    def set_x(self ,x):
+        noting = None
```
Comment (ron-popov) [inline Modules/Magnetorquer/fake_magnetorquer.py:4]: how about ```pass``` ?

File: Modules/Magnetometer/fake_magnetometer.py [hunk 2]
```diff
@@ -1,6 +1,15 @@
+from ..Field import circular_field
```
Comment (ron-popov) [inline Modules/Magnetometer/fake_magnetometer.py:1]: It seems like too much and i think the sole purpose of ```fake_magnetometer``` is to prevent the program from crashing when running the program without the magnetometer connected.  
It also means that magnetometer depends on the ```Field``` module and that is not optimal in my opinion.

File: Modules/Control/Control.py [hunk 3]
```diff
@@ -9,24 +13,37 @@
...
+        while(True):
+            arr = setControl(mode = mode)
+            self.magnetorquer.set_x(arr[0])
+            self.magnetorquer.set_y(arr[1])
+            self.magnetorquer.set_z(arr[2])
+            time.sleep(config.TIME_DIFFERENCE)
+    #TODO: Add offset for maneuvering to certain locations on earth
     def set_control(self, mode=1, offset=[0,0]):
-        #makes the direction change for the measured depolar magnetic field (cos function value)
+        # makes
```
Comment (ron-popov) [inline Modules/Control/Control.py:34]: ?

File: Modules/Field/angle.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from . import config
+'''
+Input - the time passed from measurement start
+Output - the ratio of the route that the satellite has already passed
```
Comment (ron-popov) [inline Modules/Field/angle.py:4]: "The ratio of the route" ?
Can you rephrase that a bit ?

File: Modules/Control/Control.py [hunk 5]
```diff
@@ -1,4 +1,8 @@
-from ..Magnetometer import magnetometerMT
+from ..Magnetometer import magnetometer_factory
```
Comment (ron-popov) [inline Modules/Control/Control.py:1]: Maybe you should write it as an application and not as a module ?

**Review by ron-popov — CHANGES_REQUESTED** [review]

File: Modules/Magnetometer/fake_magnetometer.py (hunk 2 above)
Comment (shaiTheKimhi) [reply Modules/Magnetometer/fake_magnetometer.py:1]: But you want is that it will return noly zero's, and we would not be able to work when the magnetometer is not connected

File: Modules/Field/angle.py (hunk 4 above)
Comment (yotam180) [reply Modules/Field/angle.py:4]: I believe he meant the quotient of the length the satellite has already traveled to the entire trajectory length.

Comment (shaiTheKimhi) [reply Modules/Field/angle.py:4]: You can also see that as "The proportion of the satellite orbit that was completed"

Comment (yotam180) [reply Modules/Field/angle.py:4]: To [ complete the sentence ]

Proportion has to consist of two measures

Comment (shaiTheKimhi) [reply Modules/Field/angle.py:4]: So percentage,
or the proprtion of completed orbit out of the entire orbit.

Comment (yotam180) [reply Modules/Field/angle.py:4]: :+1:
